src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationTestFixture.cs

```diff
@@ -138,10 +265,48 @@ protected virtual void ConfigureWebHost(IWebHostBuilder builder)
         {
         }
 
+        /// <summary>
+        /// Gives a fixture an opportunity to configure the application before it gets built.
+        /// </summary>
+        /// <param name="configure">The <see cref="Action{IWebHostBuilder}"/> to configure the <see cref="IWebHostBuilder"/>.</param>
+        public void ConfigureWebHost(Action<IWebHostBuilder> configure)
```

Comment:
This method is to allow people to manually instantiate webapplicationfixture per test and be able to do per test setup, like setting the database to a specific state that can be used later on the test.